src/components/TestimonialsCards/index.js

```diff
@@ -0,0 +1,81 @@
+import React from "react";
+import ReactStars from "react-rating-stars-component";
+
+const TestimonialsCards = () => {
+  const testimonials = [
+    {
+      rating: 5,
+      person: "Fabiana Silva – Blogueira de moda",
+      recommendationTitle: "RECOMENDO DEFINITIVAMENTE",
+      recommendationText:
+        "O impulsiona seguidores foi muito importante no processo de evolução da minha marca no mercado digital! A empresa permitiu que eu ganhasse rapidamente novos seguidores todos os dias e com muito sucesso. Graças ao impulsiona seguidores, consegui construir minha presença no Instagram de forma segura e confiável!",
+      chosenPlan: "Plano de 3000 Seguidores/Mês",
+    },
+    {
+      rating: 4,
+      person: "Nicole Andrade – Influencer Digital",
+      recommendationTitle: "ESTOU MUITO SATISFEITA",
+      recommendationText:
+        "Este é realmente um site incrível, você pode obter até 200 seguidores e 200 curtidas em um post recente todos os dias. Assinei o combo e hoje e o plano que assinei faz exatamente o que diz 100% garantido.",
+      chosenPlan: "Plano de 6000 Seguidores/Mês",
+    },
+    {
+      rating: 5,
+      person: "Fábio Oliveira – Fotógrafo",
+      recommendationTitle: "MELHOR CUSTO BENEFÍCIO",
+      recommendationText:
+        "Minhas fotos não estavam recebendo o amor que eu realmente sentia que mereciam. Sou fotógrafo de natureza há quase 2 anos, mas meu perfil nunca foi muito popular. Depois de assinar o Impulsiona Seguidores por cerca de 2 semanas, ganhei cerca de 700 novos seguidores e recebo mais 500 curtidas em média por foto e os números só tem aumentado desde então.",
+      chosenPlan: "Plano Combo 1500 Seguidores/Mês",
+    },
+    {
+      rating: 4,
+      person: "Eliane Ribeiro – Estética de Beleza",
+      recommendationTitle: "NADA A RECLAMAR",
+      recommendationText:
+        "Site absolutamente incrível, realmente funciona. Verifiquei a maioria dos perfis dos quais estou recebendo curtidas e realmente são curtidas reais de perfis fortes do Instagram.",
+      chosenPlan: "Plano 100 Curtidas/Post",
+    },
+  ];
```

Comment:
This could be set in another file. Maybe even a json. It would leave the component's file cleaner while also making it easier for us to update them later.

Reply:
Fixed in da66c2f